### DISCUSSION TITLE
Visualizations to use for the community

### DISCUSSION THREAD

**Discussion by ruudvdent** [WAM2layers/WAM2layers#221] [Show and tell]
Please show off any interesting visualizations that could be used by others in the community. If there is interest we can ship it with the code.

**Comment by ruudvdent** [comment]
@Peter9192 , could you indicate which folder in this repository would eventually be most logical to store visualization codes from the community

**Reply by Peter9192** [reply]
I can think of two options:

1. A python notebook showcasing the analysis/visualization. These would fit very well in a `notebooks` folder in the top level of the repo. I made one in https://github.com/WAM2layers/WAM2layers/pull/201, but that's not merged yet.
2. A function for a standardized figure or statistic/metric/indicator/... computed on the output. This could be part of https://github.com/WAM2layers/WAM2layers/tree/main/src/wam2layers/analysis, so other people can easily import them like `from wam2layers.analysis import fancy_plot` and then use them directly on their own output.

I can imagine a process that always starts with option 1. From there, if there is a clear benefit or interest in moving it to the option 2, we can help in converting the code to a reusable function that's part of the wam2layers core package.

**Reply by ruudvdent** [reply]
That seems to be a good way to go, but maybe `notebooks` is not the best term. I mean, it focuses on the format rather on what it does. If somebody wrote a py-script or even a R-script that should equally be possible to share. Something like: `post-processing collection`, although a bit long is a more logical name.

**Reply by Peter9192** [reply]
Perhaps just `examples` then?

**Reply by ruudvdent** [reply]
OK :)
